pages/partials/question.ejs

```diff
@@ -116,9 +122,15 @@
         This question is complete and cannot be answered again.
         <% } else { %>
         <div class="row">
-          <div class="col-sm-6">
-            <% if (showGradeButton) { %><button class="btn btn-primary question-grade">Grade</button><% } %>
-          </div>
+            <div class="col-sm-6">
+                <% if (showGradeButton) { %>
+                    <% if (question_context == 'student_homework') { %>
+                        <button class="btn btn-primary question-grade" name="__action" value="grade">Grade <span class="badge"><%= String(assessment_question.tries_per_variant - variant.num_tries) + ( ((assessment_question.tries_per_variant - variant.num_tries) > 1) ? ' tries left' : ' try left' ) %></span></button>
```

Comment:
Can we make it so that if a question doesn't have multiple attempts possible (e.g., all existing HWs) then we don't display `1 try left`?

Reply:
Sure, easy.